go.mod

```diff
@@ -25,4 +25,3 @@ require (
 	gopkg.in/ini.v1 v1.42.0 // indirect
 	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787
 )
-
```

Comment:
Could you please revert the removal of the trailing empty line? I have some local scripts that conflict with this :(